app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1530,6 +1530,11 @@ public void finishWidgetMove() {
         queueRunnable(this::finishWidgetMoveNative);
     }
 
+    @Override
+    protected void showIncomingMessageNotification(String message) {
+        mNavigationBar.showIncomingMessageNotification(message);
+    }
```

Comment:
This seems wrong. First of all we should only have this code in HVR. Secondly I doubt this method overrides anything...